install.py

```diff
@@ -104,7 +75,13 @@ def get_release_for_platform(releases, platform):
     return None
 
 
+def has_bin_in_path(bin_path):
+    """Determine if the binary path is part of the system paths."""
+    return '{}:'.format(bin_path) in os.environ['PATH']
```

Comment:
Hmm, this wouldn't resolve to true if they set their path like:

```
export PATH=$PATH:$HOME/bin
```

Reply:
Maybe something like:
```
return bin_path in os.environ['PATH'].split(':')
```

Reply:
Thanks, updated.